src/components/tasks/card/card.module.scss

```diff
@@ -103,6 +104,10 @@ $seeMoreTasksShadow: #595959cc;
 .cardStrongFont {
     font-size: 1.3rem;
     font-weight: 500;
+    border-radius: 5px;
+    background-color: $grey;
+    width: 7rem;
+    height: 1.5rem;
 }
 
```

Comment:
same as above follow the design guidelines and stick to rem or em or px

Reply:
Again, i think it is not necessary to stick to one unit, we have to consider if it needs scaling up or down. I do not want the border-radius to scale up/down that is why it is in absolute unit like px.